Dockerfile

```diff
@@ -1,36 +1,28 @@
-FROM python:3.10-buster
+FROM ubuntu:22.04
 LABEL GeoNode development team
 
 RUN mkdir -p /usr/src/geonode
 
-# Enable postgresql-client-13
-RUN apt-get update -y && apt-get install curl wget unzip -y
-RUN echo "deb http://apt.postgresql.org/pub/repos/apt/ buster-pgdg main" | tee /etc/apt/sources.list.d/pgdg.list
-RUN echo "deb http://deb.debian.org/debian/ stable main contrib non-free" | tee /etc/apt/sources.list.d/debian.list
+## Enable postgresql-client-13
+RUN apt-get update -y && apt-get install curl wget unzip gnupg2 -y
 RUN wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
-
-# To get GDAL 3.2.1 to fix this issue https://github.com/OSGeo/gdal/issues/1692
-# TODO: The following line should be removed if base image upgraded to Bullseye
-RUN echo "deb http://deb.debian.org/debian/ bullseye main contrib non-free" | tee /etc/apt/sources.list.d/debian.list
-
-# This section is borrowed from the official Django image but adds GDAL and others
-RUN apt-get update -y && apt-get upgrade -y
+# will install python3.10 
+RUN apt-get install lsb-core -y
+RUN echo "deb http://apt.postgresql.org/pub/repos/apt/ `lsb_release -cs`-pgdg main" |tee  /etc/apt/sources.list.d/pgdg.list
 
 # Prepraing dependencies
 RUN apt-get install -y \
     libgdal-dev libpq-dev libxml2-dev \
     libxml2 libxslt1-dev zlib1g-dev libjpeg-dev \
     libmemcached-dev libldap2-dev libsasl2-dev libffi-dev
 
-RUN apt-get install -y --no-install-recommends \
+RUN apt-get update -y && apt-get install -y --no-install-recommends \
     gcc zip gettext geoip-bin cron \
     postgresql-client-13 \
-    sqlite3 spatialite-bin libsqlite3-mod-spatialite \
     python3-all-dev python3-dev \
     python3-gdal python3-psycopg2 python3-ldap \
-    python3-pip python3-pil python3-lxml python3-pylibmc \
```

Comment:
I had to remove `pylibmc` because was raising this issue:
```
pip cannot uninstall pylibmc: "It is a distutils installed project"
```

Now it get installed via pip and no longer via the apt-get